web/client/selectors/mapsave.js

```diff
@@ -61,7 +62,8 @@ export const basicMapOptionsToSaveSelector = createStructuredSelector({
     }),
     featureGrid: createStructuredSelector({
         attributes: customAttributesSettingsSelector
-    })
+    }),
+    swipe: getSwipeSettingsSelector
```

Comment:
```suggestion
```
Avoid modifying the `basicMapOptionsToSaveSelector`. Instead use `registerCustomSaveHandler` to register custom props to save.